rules/lowercase_description.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+const ruleMsg = 'it() description should begin with lowercase';
+
+const isItOrTestorDescribeFunction = node => {
+  return (
+    node.type === 'CallExpression' &&
+    node.callee &&
+    (node.callee.name === 'it' ||
+      node.callee.name === 'test' ||
+      node.callee.name === 'describe')
+  );
+};
+
+const isItDescription = node => {
+  const foo =
+    node.arguments &&
+    node.arguments[0] &&
+    (node.arguments[0].type === 'Literal' ||
+      node.arguments[0].type === 'TemplateLiteral');
+  return foo;
+};
+
+const testDescription = node => {
+  const { type } = node.arguments[0];
+  if (type === 'Literal') {
+    return node.arguments[0].value;
+  }
+  if (type === 'TemplateLiteral') {
+    return node.arguments[0].quasis[0].value.raw;
+  }
+};
+
+const descriptionBeginsWithLowerCase = node => {
+  if (isItOrTestorDescribeFunction(node) && isItDescription(node)) {
+    const description = testDescription(node);
+    return description[0] === description[0].toLowerCase();
+  }
+  return true;
+};
+
+module.exports = {
+  meta: {
+    docs: {
+      url:
+        'https://github.com/jest-community/eslint-plugin-jest/blob/master/docs/rules/lowercase_description.md',
```

Comment:
`lowercase_description` should be `lowercase-description` to match the `lowercase-description.md` filename.

I think if you have master merged in, this is autofixable with ESLint (introduced in #60).

Reply:
I did have the merged changes. The issue was that the file name was `rules/lowercase_description.js`, instead of `rules/lowercase-description.js`.